### PR TITLE
Public website

### PR DESCRIPTION
**Pull request by erunion**
This moves all of the content that we have documented in the [wiki](https://github.com/vimeo/mill/wiki) into a public website that, built with Jekyll, will be hosted on Github Pages.

It looks nice!

fullscreen_7_27_17__11_09_pm

fullscreen_7_27_17__11_12_pm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Mill
+&#9780; Mill
```
Comment (agilefox) [inline README.md]: What is &#9780?

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# site settings
+title: "☴ Mill"
```
Comment (agilefox) [inline docs/_config.yml]: Why the TRIGRAM FOR WIND?

Comment (agilefox) [inline docs/_config.yml]: I mean, if it's just style, cool ... just not sure if that's a Jeckyll or githubpages thing.

File: docs/_layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    <section class="nav">
+        <nav>
+            <ul>
+                <li><a href="/" id="logo">☴ Mill</a></li>
+                <li><a href="/getting-started">Getting Started</a></li>
+                <li><a href="/configuration">Configuration</a></li>
+                <li>
+                    <a href="/generate">Generate</a>
+                    <ul>
+                        <li><a href="/generate/documentation">&middot; Documentation</a></li>
```
Comment (agilefox) [inline docs/_layouts/default.html]: Why doesn't the style for `<li>` give you the `&middot;`. Why the specific request?

File: README.md (hunk 1 above)
Comment (erunion) [inline README.md]: ASCII for ☴.

File: docs/_config.yml (hunk 2 above)
Comment (erunion) [inline docs/_config.yml]: ☴ is the new Mill logo/symbol.

File: docs/_layouts/default.html (hunk 3 above)
Comment (erunion) [inline docs/_layouts/default.html]: Laziness mostly.

**Pull request merged**